apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -303,15 +303,15 @@ public EmulatorDetection EmulatorDetection
             get
```

Comment:
nit: get and set here are not consistent. set fails if the value is out of range but get returns None.

Reply:
That's the same for all the properties. For int ones for example, we have `GetInt32OrDefault`, but `SetInt32WithValidation`.